numba_dppy/tests/njit_tests/dpnp/__init__.py

```diff
@@ -16,4 +16,10 @@
 # limitations under the License.
 ################################################################################
 
+import pytest
+
+from numba_dppy.tests._helper import ensure_dpnp
+
 from . import *
+
+pytestmark = pytest.mark.xfail(not ensure_dpnp(), reason="No dpnp available")
```

Comment:
Why xfail?

Reply:
Our goal make all tests with import dpnp problem xfail
And besides, if use a skip, then the first test in any case falls from the numba error, I don't know why

Reply:
Interesting. So xfail hides a problem.